### PR TITLE
add apt-get.json

### PR DESCRIPTION
**Pull request by abadojack**
**What does your Instant Answer do?**
My instant answer provides the most commonly used commands of the AptGet package manager for Debian based GNU/Linux systems.

**What is the data source for your Instant Answer? (Provide a link if possible)**
[Ubuntu Wiki](https://help.ubuntu.com/community/AptGet/Howto)

**Why did you choose this data source?**
Ubuntu Wiki has the most comprehensive documentation of the AptGet package managet

**Are there any other alternative (better) data sources?**
[Debian Wiki](https://wiki.debian.org/Apt)

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
GNU/Linux users

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
No

**Are you having any problems? Do you need our help with anything?**
No

**Where did you hear about DuckDuckHack? (For first time contributors)**
I heard about it from an Instant answer.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
It's a cheat sheet which shows AptGet commands

-----
IA Page: https://duck.co/ia/view/apt_get_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@abadojack Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

File: share/goodie/cheat_sheets/json/apt-get.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        "Installation commands",
+        "Maintenance commands",
+        "Removal commands",
+        "Search commands"
+    ],
+    "sections": {
+        "Installation commands": [
+            {
+                "key": "Install a new package",
+                "val": "apt-get install <package_name>"
```
Comment (codenirvana) [inline share/goodie/cheat_sheets/json/apt-get.json]: Terminal command should be the ```key``` not the ```val```.

**Comment by moollaza** [comment]
@abadojack thanks for your contribution!

Can you please fill out the [pull request template](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/pull_request_template_cheat_sheet.md) and edit your initial comment. Once completed, we'll be sure to review your PR and give you some feedback.

Thanks!

File: share/goodie/cheat_sheets/json/apt-get.json (hunk 1 above)
Comment (moollaza) [inline share/goodie/cheat_sheets/json/apt-get.json]: @abadojack as @codenirvana point out, you need to swap the content for all your `key` and `val` properties.

**Comment by abadojack** [comment]
I have filled the pull request template and swapped contents of key and val

**Comment by moollaza** [comment]
> I have filled the pull request template and swapped contents of key and val

@abadojack Perfect! I've setup the Instant Answer Page to be created and then this will be deployed to beta.duckduckgo.com for testing

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Apt get cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/apt_get_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
@abadojack I'm very sorry for the delay on this, it fell through the crack. I'll try to make sure it doesn't happen again!

This LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by moollaza** [comment]
@abadojack congrats! It's live: https://duckduckgo.com/?q=Apt+get+cheat+sheet&t=ffcm&ia=answer

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Comment by abadojack** [comment]
@moollaza nice. That's a beautiful cheat sheet.

**Pull request merged**